src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -1549,6 +1549,13 @@ bool ConnectivityManagerImpl::_GetBssInfo(const gchar * bssPath, NetworkCommissi
     std::unique_ptr<GVariant, GVariantDeleter> ssid(g_dbus_proxy_get_cached_property(G_DBUS_PROXY(bssProxy), "SSID"));
     std::unique_ptr<GVariant, GVariantDeleter> bssid(g_dbus_proxy_get_cached_property(G_DBUS_PROXY(bssProxy), "BSSID"));
 
+    // Network scan is performed in the background, so the BSS
+    // may be gone when we try to get the properties.
+    if (ssid == nullptr || bssid == nullptr)
+    {
+        return false;
```

Comment:
should we log something if this happens? Although there is no logging on null bss either.

Reply:
Log message added.

> Although there is no logging on null bss either.

NULL bss would be caused out-of-memory (I think) or some programmer error. The `wpa_fi_w1_wpa_supplicant1_bss_proxy_new_for_bus_sync` only creates and initializes GLib object.